api/python/cellxgene_census/tests/experimental/pp/test_hvg.py

```diff
@@ -209,7 +209,7 @@ def test_hvg_vs_scanpy(
             "Homo sapiens",
             "is_primary_data == True",
             "dataset_id",
-            slice(500_000, 1_000_000),
+            slice(1_000_000, 4_000_000),
```

Comment:
the latest LTS census has no primary cells in the [500_000,1_000_000] soma joniid range, so this test will fail.  Expanded the range to ensure we pick up some cells.